eachdist.ini

```diff
@@ -34,6 +34,7 @@ version=0.39b0.dev
 
 packages=
     opentelemetry-opentracing-shim
+    opentelemetry-opencensus-shim
```

Comment:
We exclude this shim from `build.sh`. This is added here to update the `version.py`.

Reply:
As in, we didn't have this here before and that was a bug?

Reply:
Not really a bug since we excluded this knowingly earlier. Adding it here only updates the version to keep it in sync with the rest of the packages. We don't release this yet so it's not an issue.